javalanglib/src/main/scala/java/lang/Math.scala

```diff
@@ -4,25 +4,59 @@ package lang
 import scala.scalajs.js
 
 object Math {
-  final val E  = 2.718281828459045
+  final val E = 2.718281828459045
   final val PI = 3.141592653589793
 
-  @inline def abs(a: scala.Int): scala.Int = if (a < 0) -a else a
-  @inline def abs(a: scala.Long): scala.Long = if (a < 0) -a else a
-  @inline def abs(a: scala.Float): scala.Float = js.Math.abs(a).toFloat
-  @inline def abs(a: scala.Double): scala.Double = js.Math.abs(a)
+  @inline def sin(a: scala.Double): scala.Double = js.Math.sin(a)
```

Comment:
I understand. The boy scout rule is good to apply unless it makes a diff too hard to read. In this case, yes, please preserve the old ordering, and just append the new methods at the end.